wireless_modules/middle_module/upload.sh

```diff
@@ -1,25 +1,71 @@
 #!/bin/sh
-ampy -p $1 -b 115200 put ../config.py
-echo "config.py loaded"
-ampy -p $1 -b 115200 put main.py
-echo "main.py loaded"
-ampy -p $1 -b 115200 put ../boot.py
-echo "boot.py loaded"
-ampy -p $1 -b 115200 put ../mqtt_client.py
-echo "mqtt_client.py loaded"
-ampy -p $1 -b 115200 put ../wireless_module.py
-echo "wireless_module.py loaded"
-ampy -p $1 -b 115200 put ../libraries/MPU6050-ESP8266-MicroPython/mpu6050.py
-echo "mpu6050.py loaded"
-ampy -p $1 -b 115200 put ../sensors/mpu.py
-echo "mpu.py loaded"
-ampy -p $1 -b 115200 put ../sensors/dht_sensor.py
-echo "dht_sensor.py loaded"
-ampy -p $1 -b 115200 put ../sensors/co2_sensor.py
-echo "mq135_sensor.py loaded"
-ampy -p $1 -b 115200 put ../libraries/MQ135/mq135.py
-echo "mq135.py loaded"
-ampy -p $1 -b 115200 put ../sensors/sensor_base.py
-echo "sensor_base.py loaded"
-ampy -p $1 -b 115200 put ../libraries/abc.py
-echo "abc.py loaded"
+
+# Purpose: To upload specific files or all relevant middle wireless module files (default) onto the ESP32
+# Requirements: Must have ampy downloaded
+
+# NOTE: 
+# Shell scripting should have no white space between the '=' operator for variable assigning
+
+# declares an integer
+declare -i BAUD_RATE=115200
+
+PORT=$1
+NUM_ARGS=$#
+files="main.py ../config.py ../wireless_module.py  ../boot.py ../mqtt_client.py ../sensors/mpu.py \
```

Comment:
Can this be stored in an array of some sort, ideally with one file per line? This will make git play much more nicely with this file, as it's likely that this file is frequently edited and likely to cause conflicts.

Reply:
When I first searched I couldn't find array types in shell, but I looked again and apparently it is possible, so it shall be refactored!